### ISSUE TITLE
CI error on MacOS: argument "force" is missing, with no default

### ISSUE THREAD

**Issue by ahmohamed** [Bioconductor/BiocManager#107]
I'm seeing this error on my CI:

```
Bioconductor version 3.13 (BiocManager 1.30.15), R 4.1.0 (2021-05-18)
Installing package(s) 'BiocVersion'
trying URL 'https://bioconductor.org/packages/3.13/bioc/bin/macosx/contrib/4.1/BiocVersion_3.13.1.tgz'
Content type 'application/x-gzip' length 5574 bytes
==================================================
downloaded 5574 bytes


The downloaded binary packages are in
Error in .install_repos(pkgs, old_pkgs, instPkgs = instPkgs, lib = lib,  : 
	/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T//RtmpnvXnRT/downloaded_packages
  argument "force" is missing, with no default
Calls: <Anonymous> ... .install -> .install_repos -> .install_filter_up_to_date
Execution halted
```

The error seems to be related to this commit https://github.com/Bioconductor/BiocManager/commit/af0aaadf68b92f16e8888f1fe0c3619b3ad9303f but I'm not sure how to fix. Other platforms don't seem to have this issue.

CI results: https://github.com/ahmohamed/lipidr/runs/2757603166#step:13:1

Cheers,
Ahmed.

**Comment by mbstadler** [comment]
We see the same issue, but not only on macOS but also on:
- Windows, e.g. at https://github.com/fmicompbio/eisaR/runs/2763471903?check_suite_focus=true
- Linux, e.g. at https://github.com/csoneson/alevinQC/runs/2744796828?check_suite_focus=true

**Comment by csoneson** [comment]
I can reproduce it also locally with Bioc 3.14. It seems it may come from  https://github.com/Bioconductor/BiocManager/commit/d0b94b94236bc0014146dedd4064bfb770abb065 - perhaps `checkBuilt` should to be removed from [here](https://github.com/Bioconductor/BiocManager/blob/a88deb0f39bd92d94f0d9c6569ca13a13e8c1968/R/install.R#L197-L200)?

**Comment by LiNk-NY** [comment]
This happens immediately after `BiocVersion` is installed. Can you try testing with the `pass-force` branch? https://github.com/bioconductor/biocmanager/tree/pass-force
```
BiocManager::install("Bioconductor/BiocManager", ref = "pass-force")
```

**Comment by csoneson** [comment]
Thank you Marcel! It looks like that solves it on my side.

**Comment by LiNk-NY** [comment]
Note. This bug can also be seen here: https://github.com/Bioconductor/bioconductor_docker/issues/30

**Comment by LiNk-NY** [comment]
Hi, 
This should have been resolved here: 205071a9d06064a53da58f208d9579355e668f30
Best, 
Marcel
